RecoTracker/FinalTrackSelectors/python/preDuplicateMergingGeneralTracks_cfi.py

```diff
@@ -18,7 +18,7 @@
         cms.InputTag("muonSeededTracksInOutSelector","MVAVals"), 
         cms.InputTag("muonSeededTracksOutInSelector","MVAVals"), 
     ),
-    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(False), tLists = cms.vint32(0, 1,2))),
+    setsToMerge = cms.VPSet(cms.PSet(pQual = cms.bool(True), tLists = cms.vint32(0, 1,2))),
```

Comment:
there should be no changes in physics performance at this point in the release cycle.
Is the impact of this line change in line with what we observe in validation plots?

(if it's the origin, can this be just changed back to False?)